main/options.c

```diff
@@ -2417,7 +2417,7 @@ static parametricOption ParametricOptions [] = {
 	{ "_list-roles",            processListRolesOptions,        TRUE,   STAGE_ANY },
 	{ "maxdepth",               processMaxRecursionDepthOption, TRUE,   STAGE_ANY },
 	{ "options",                processOptionFile,              FALSE,  STAGE_ANY },
-	{ "output-format",          processOutputFormat,            TRUE,   STAGE_ANY},
+	{ "output-format",          processOutputFormat,            TRUE,   STAGE_ANY },
```

Comment:
I wonder why this change is needed. All entries has a whitespace after STAGE_ANY.